packages/rs-drive-abci/src/execution/platform_events/withdrawals/append_signatures_and_broadcast_withdrawal_transactions/v0/mod.rs

```diff
@@ -80,12 +89,39 @@ where
                         index,
                         e
                     );
-
-                    return Err(e.into());
+                    // These errors might allow the state transition to be broadcast in the future
+                    transaction_submission_failures.push((transaction.txid(), tx_bytes));
                 }
             }
         }
 
+        store_transaction_failures(transaction_submission_failures)
+            .map_err(|e| Error::Execution(e.into()))?;
+
         Ok(())
     }
 }
+
+// Function to handle the storage of transaction submission failures
+fn store_transaction_failures(failures: Vec<(Txid, Vec<u8>)>) -> std::io::Result<()> {
+    // Ensure the directory exists
+    let dir_path = "transaction_submission_failures";
```

Comment:
This should be config option, or at least relative to some config option

Reply:
@lklimek I agree, let's do this in a different PR tomorrow.